ex/wasm/hello-wasm/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>hello-wasm example</title>
+</head>
+
+<body>
+    <script type="module">
+        import init, { greet } from "./pkg/hello_wasm.js";
```

Comment:
TODO: this directory doesn't have a README.md and I already forget how to build it :D